source/includes/_simulator-guide.html.md

```diff
@@ -0,0 +1,130 @@
+
+The `Config` object has a built in command line parser which can be used to override the local running environment. It comes with a set of standard switches which can be viewed with the `--help` option. The current default configuration is read from environment variables, a `~/.bonsai` configuration file in the user’s home directory, a `./.bonsai` and a `./.brains` configuration file in the current working directory, command line options, and programmatically in code.
+
+During the course of normal operation, one should only need to specify the prediction/training mode on the command line. The rest of the options will be read from configuration files.
+
+One is not limited to launching a single simulator locally. If you have the compute resources, each new simulator will open a new connection to the server and add to the training pool. You can do this on a per process basis, or at the thread level.
+
+
+## Proxy Support
+> Example Python Proxy
+
+```python
+my_config.proxy == 'myproxy:5000'
+```
+
+> Example CLI Proxy Usage
+
+```shell
+$ python <simulator_file>.py --proxy=localhost:3128
+```
+
+To connect a simulator from behind a corporate proxy, command line tools like the Bonsai SDK need to know where to send traffic. For the Bonsai SDK, this is done by providing the proxy server address and port to the simulator config when running a simulator. There are multiple ways that you can do this.
+## Prediction
+
+```python
+# Default version is latest
+python <simulator_file>.py --predict
+
+#OR
+Python <simulator_file>.py --predict=1
+```
+
+After your BRAIN is finished training it can be used to "predict" or perform in the simulation. How well it does depends on how long you let it train! Using your BRAIN involves running your simulator file as you did when training, but now in prediction mode with `--predict`. The default is `--predict=latest` which will use the version of the latest training session that you just ran. You can use a different version of your BRAIN if you have trained it multiple times by replacing `latest` with the version number.
+
+# Cloud Hosted Simulators
+There are a couple simulators that the Bonsai Platform can run cloud-hosted (sometimes referred to as platform managed) in a Docker container. This means you can start and stop training all from the web interface, without needing to download and install the Bonsai CLI or SDK. You will still need to run the simulator locally using the CLI to obtain predictions, however.
+
+Cloud-hosted simulators are useful for prolonged training to avoid your computer becoming idle and stopping training. At this time, you can only run one simulator cloud-hosted, and cannot take advantage of training speed increases from [running simulators in parallel][5].
+
+Current list of supported simulators for Docker cloud-hosted training:
+
+* OpenAI Gym environment: [`openai.gym`][1]
```

Comment:
not all gym environments work. e.g. lunarlander doesn't.

Reply:
The gym-common library is just a library. This isn't trying to say that we support every OpenAI Gym Environment out of the box. I changed it to "Current list of supported libraries". Does that work?